operator/node.go

```diff
@@ -156,7 +155,9 @@ func (n *operatorNode) Start(logger *zap.Logger) error {
 	go n.reportOperators(logger)
 
 	go n.feeRecipientCtrl.Start(logger)
-	n.dutyCtrl.Start(logger)
+	if err := n.dutyScheduler.Run(n.context, logger); err != nil {
+		return errors.Wrap(err, "failed to run duty scheduler")
```

Comment:
Let's not use `"github.com/pkg/errors"` as it's discontinued
```suggestion
		return fmt.Errorf("run duty scheduler: %w", err)
```